README.md

```diff
@@ -1,53 +1,52 @@
+[![awesome-runners](https://img.shields.io/badge/listed%20on-awesome--runners-blue.svg)](https://github.com/jonico/awesome-runners)
 
-## Quick-Start (Examples and Usage) ##
+This repository facilitates running the [self-hosted GitHub Actions runners](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/hosting-your-own-runners).
 
-Please see [the wiki](https://github.com/myoung34/docker-github-actions-runner/wiki/Usage)
-Please read [the contributing guidelines](https://github.com/myoung34/docker-github-actions-runner/blob/master/CONTRIBUTING.md)
+## Quick Start 
 
-## Notes ##
+- [Examples and Usage](https://github.com/myoung34/docker-github-actions-runner/wiki/Usage)
+- [Contributing Guidelines](https://github.com/myoung34/docker-github-actions-runner/blob/master/CONTRIBUTING.md)
 
-### Security ###
+## Notes
 
-It is known that environment variables are not safe from exfiltration.
-If you are using this runner make sure that any workflow changes are gated by a verification process (in the actions settings) so that malicious PR's cannot exfiltrate these.
+### Security
 
-### Docker Support ###
+Please be aware that environment variables are susceptible to exfiltration. When using this runner, ensure that any workflow changes undergo a verification process (within the actions settings) to prevent malicious PRs from exfiltrating this information.
 
-Please note that while this runner installs and allows docker, github actions itself does not support using docker from a self hosted runner yet.
-For more information:
+### Docker Support
 
-* https://github.com/actions/runner/issues/406
-* https://github.com/actions/runner/issues/367
+Although this runner installs and supports Docker, GitHub Actions currently does not support using Docker with a self-hosted runner. For more details:
+- [Issue #406](https://github.com/actions/runner/issues/406)
+- [Issue #367](https://github.com/actions/runner/issues/367)
 
-Also, some GitHub Actions Workflow features, like [Job Services](https://docs.github.com/en/actions/guides/about-service-containers), won't be usable and [will result in an error](https://github.com/myoung34/docker-github-actions-runner/issues/61).
+Additionally, certain GitHub Actions Workflow features, like [Job Services](https://docs.github.com/en/actions/guides/about-service-containers), are not supported and [may lead to errors](https://github.com/myoung34/docker-github-actions-runner/issues/61).
 
-### Containerd Support ###
+### Containerd Support
 
-Currently runners [do not support containerd](https://github.com/actions/runner/issues/1265)
+As of now, runners [do not support containerd](https://github.com/actions/runner/issues/1265).
 
-## Docker Artifacts ##
+## Docker Artifacts
 
 | Container Base | Supported Architectures | Tag Regex | Docker Tags | Description | Notes |
-| --- | --- | --- | --- | --- | --- |
+|:--------------:|:-----------------------:|:---------:|:-----------:|:-----------:|:-----:|
```

Comment:
what are these colons?there's no need to change spacing, markdown is smart enough to know that pipes and 3 dashes is enough